deploy/charts/harvester/values.yaml

```diff
@@ -374,7 +374,7 @@ harvester-node-disk-manager:
     repository: rancher/harvester-node-disk-manager
     pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
-    tag: v0.7.2
+    tag: v0.7.3
```

Comment:
Hi @brandboat,
Sorry for that, but we need to bump to v0.7.4 (image tag) for NDM

Reply:
Thanks, addressed in [dfde93d](https://github.com/harvester/harvester/pull/6654/commits/dfde93d493e2de9170087821ed60b8aa19b9175c)

Reply:
Thank you!